client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-how-to-migrate/index.tsx

```diff
@@ -0,0 +1,121 @@
+import { Card } from '@automattic/components';
+import { StepContainer, SubTitle, Title } from '@automattic/onboarding';
+import { getQueryArg } from '@wordpress/url';
+import { useTranslate } from 'i18n-calypso';
+import DocumentHead from 'calypso/components/data/document-head';
+import { useAnalyzeUrlQuery } from 'calypso/data/site-profiler/use-analyze-url-query';
+import { useHostingProviderQuery } from 'calypso/data/site-profiler/use-hosting-provider-query';
+import { useSite } from 'calypso/landing/stepper/hooks/use-site';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { usePresalesChat } from 'calypso/lib/presales-chat';
+import useHostingProviderName from 'calypso/site-profiler/hooks/use-hosting-provider-name';
+import type { Step } from '../../types';
+
+import './style.scss';
+
+const SiteMigrationHowToMigrate: Step = function ( { navigation } ) {
+	const translate = useTranslate();
+	const site = useSite();
+	const importSiteQueryParam = getQueryArg( window.location.href, 'from' )?.toString() || '';
+
+	const options = [
```

Comment:
```suggestion
	const options = useMemo(()=> ....
```
A new instance array instance  is recreated in every render and causes screen rerenders on every internal change.
Use memo helps us to avoid it.